packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -177,6 +178,7 @@ class UnThemedLogRow extends PureComponent<Props, State> {
             updateLimit={updateLimit}
             context={context}
             showContext={showContext}
+            showContextToggle={true}
```

Comment:
This needs to be changed as currently it means the "Show Context" label is displayed at all times, which isn't the desired behavior for Loki

Reply:
Do you mean we need to be able to toggle "show context" as a feature? ie disable it for datasource that do not support it? Cause I think we should be already doing that, don't think the context feature is shown for Elastic for example.

Reply:
So currently the logic in `LogRowMessage.tsx` is such that the context toggle is only displayed if `(row.searchWords && row.searchWords.length > 0)`, so I added `showContextToggle` to have more control over when it is or isn't shown